tools/metrics/aiven.go

```diff
@@ -91,6 +90,7 @@ func AivenCostGauge(client *AivenClient, interval time.Duration) MetricReadClose
 					Time:  time.Now(),
 					Name:  "aiven.estimated.cost",
 					Value: currentCost,
+					Unit:  "pounds",
```

Comment:
I've checked the Aiven API docs and can't see a unit, here. Are we sure it's £££ not $$$?

Reply:
I've checked it in the Aiven console and the invoices are in pounds.